src/components/DeprecatedBanner/DeprecatedBanner.js

```diff
@@ -64,28 +64,27 @@ const DeprecatedBody = ({ dao }) => (
       constructed with on Rinkeby. As a result, we’ve made the tough decision to
       deprecate all old testnet organizations. Please see{' '}
       <StyledSafeLink href={DEPRECATION_URL} target="_blank">
-        {DEPRECATION_URL}
+        our blog post
       </StyledSafeLink>{' '}
       for more details.
     </TopParagraph>
     <Text.Paragraph color={textDimmed}>
-      On <time dateTime="2018-11-01">Nov. 1, 2018</time>, this organization will
-      no longer be accessible on app.aragon.one. To help migrate, and in case
-      you still need access to this organization, we will continue hosting this
-      version of Aragon Core on{' '}
-      <StyledSafeLink href={`${OLD_DAO_URL}${dao}`} target="_blank">
```

Comment:
@bpierre I think this old behaviour was still nice; it would try to open the same DAO on the old app instance in case it was available there. Now we'd just bring them to a (dead) onboarding screen.